src/newComponents/viewer/LeadPreview/ExternalUrlPreview/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useMemo } from 'react';
+import { PendingMessage } from '@the-deep/deep-ui';
+
+import { useRequest } from '#utils/request';
+import {
+    MimeTypes,
+} from '../Preview/mimeTypes';
+
+import Viewer from '../Preview';
+
+// NOTE: Need to confirm if this is necessary
+// This is only necessary if even the keys might be differently cased
```

Comment:
This was done previously for some reason. This could be directly accessed from the object, but the keys are also lowercased, which leads me to believe that sometimes we could get the keys of the header object in the random casing.

Reply:
The keys can be differently cased depending on the server.